cluster-scope/overlays/nerc-ocp-prod/ingresscontrollers/external-apps-ingress-controller.yaml

```diff
@@ -15,6 +15,6 @@ spec:
     nodeSelector:
       matchLabels:
         nerc.mghpcc.org/external-ingress: 'true'
-  namespaceSelector:
+  routeSelector:
     matchLabels:
       type: external
```

Comment:
Actually, here is one more change I think we should make. I think that `type` is probably too generic. Let's (a) pick a more descriptive name, and (b) let's namespace it like we have for other locally relevant labels.

How about:

```
routerSelector:
  matchLabels:
    nerc.mghpcc.org/external-ingress: "true"
```

This is the same label we use to identify nodes that host the external ingress address (and placement for the external ingress service).

Reply:
@larsks good points, updated the commit messages and changed the label